### PR TITLE
TFTP offers various HW architectures (jsc#SLE-10057)

### PR DESCRIPTION
**Pull request by tbazant**
### Description
The HW architecture of TFTP server can vary from that of the client machines.

### Checklist
* Check all items that apply.

*Are backports required?*

- [ ] To maintenance/SLE15SP1
- [ ] To maintenance/SLE15SP0
- [ ] To maintenance/SLE12SP5
- [ ] To maintenance/SLE12SP4
- [ ] To maintenance/SLE12SP3

### CODE REVIEW

File: xml/deployment_pxe.xml [hunk 1]
```diff
@@ -290,24 +290,59 @@ DHCLIENT6_CLIENT_ID=<replaceable>00:03:52:54:00:02:c2:67</replaceable></screen>
...
-    architectures of &productname; to the
-    <filename>/srv/tftpboot</filename> directory.
+    You can also copy the files for other versions and architectures of
+    &productname; to the <filename>/srv/tftpboot</filename> directory.
    </para>
+   <tip>
+    <title>Serving Different Architectures</title>
+    <para>
+     The client and server hardware architecture can vary. For example, you can
+     run a &x86-64; TFTP server and provide bootable environment for &aarch64;
```
Comment (lproven) [inline xml/deployment_pxe.xml:341]:
```suggestion
     run a &x86-64; TFTP server and provide a bootable environment for &aarch64;
```

File: xml/deployment_pxe.xml [hunk 2]
```diff
@@ -290,24 +290,59 @@ DHCLIENT6_CLIENT_ID=<replaceable>00:03:52:54:00:02:c2:67</replaceable></screen>
    <title>Installing Files for Boot</title>
    <para>
     &productname; provides the required files for booting via a PXE on BIOS or
```
Comment (lproven) [inline xml/deployment_pxe.xml:292]:
```suggestion
    &productname; provides the required files for booting via PXE on BIOS or
```

File: xml/deployment_pxe.xml [hunk 3]
```diff
@@ -290,24 +290,59 @@ DHCLIENT6_CLIENT_ID=<replaceable>00:03:52:54:00:02:c2:67</replaceable></screen>
...
+    <listitem>
+     <para>
+      &zseries;
+     </para>
+    </listitem>
+   </itemizedlist>
+
+   <para>
+    Files required to boot from a specific hardware architecture are included
+    in a RPM package. Install it on the machine running the TFTP server:
```
Comment (lproven) [inline xml/deployment_pxe.xml:321]:
```suggestion
    in an RPM package. Install it on the machine running the TFTP server:
```

File: xml/deployment_pxe.xml [hunk 4]
```diff
@@ -290,24 +290,59 @@ DHCLIENT6_CLIENT_ID=<replaceable>00:03:52:54:00:02:c2:67</replaceable></screen>
...
    </para>
    <screen>&prompt.sudo;<command>zypper in tftpboot-installation-SLE-<replaceable>OS_VERSION</replaceable>-<replaceable>ARCHITECTURE</replaceable></command></screen>
    <para>
     Replace <replaceable>OS_VERSION</replaceable> with the version of your
-    &productname; installation, for example <replaceable>SLE-15-SP1-x86_64</replaceable> 
-    with the architecture of your
-    system, for example <literal>x86_64</literal>. Run
-    <command>zypper se tftpboot</command> to search for all available
-    versions and architectures.
+    &productname; installation, for example
```
Comment (lproven) [inline xml/deployment_pxe.xml:326]: I think a few words were lost here (already, in the older text) and what is left is garbled.

> Replace &lt;replaceable&gt;OS_VERSION&lt;/replaceable&gt; with the version of your &productname; installation,
> for example &lt;replaceable&gt;SLE-15-SP1-x86_64&lt;/replaceable&gt; with the architecture of your system, for 
> example &lt;literal&gt;x86_64&lt;/literal&gt;.

I think this should read something like the following:

> Replace &lt;replaceable&gt;OS_VERSION&lt;/replaceable&gt; with the version number of your &productname; 
> installation, for example &lt;replaceable&gt;15-SP1&lt;/replaceable&gt;, and replace
> &lt;replaceable&gt;ARCHITECTURE&lt;/replaceable&gt; with the architecture of your system, for example 
> &lt;literal&gt;x86_64&lt;/literal&gt;. So the resulting text would look like this:
> &lt;filename&gt;tftpboot-installation-SLE-15-SP1-X86_64&lt;/filename&gt;.

I also note that there is a mixture of lower-case and upper-case in the example (x86-64 **and** X86-64) but I am not sure which is correct.

File: xml/deployment_pxe.xml (hunk 1 above)
Comment (lproven) [inline xml/deployment_pxe.xml]:
```suggestion
     run an &x86-64; TFTP server and provide a bootable environment for &aarch64;
```

File: xml/deployment_pxe.xml [hunk 5]
```diff
@@ -289,25 +289,60 @@ DHCLIENT6_CLIENT_ID=<replaceable>00:03:52:54:00:02:c2:67</replaceable></screen>
...
-    architectures of &productname; to the
-    <filename>/srv/tftpboot</filename> directory.
+    You can also copy the files for other versions and architectures of
+    &productname; to the <filename>/srv/tftpboot</filename> directory.
    </para>
+   <tip>
+    <title>Serving Different Architectures</title>
+    <para>
+     The client and server hardware architecture can vary. For example, you can
+     run a &x86-64; TFTP server and provide a bootable environment for &aarch64;
```
Comment (tbazant) [inline xml/deployment_pxe.xml:341]:
```suggestion
     run an &x86-64; TFTP server and provide a bootable environment for &aarch64;
```

File: xml/deployment_pxe.xml (hunk 1 above)
Comment (tbazant) [reply xml/deployment_pxe.xml]: fixed

**Pull request merged**